code/index.html

```diff
@@ -24,24 +24,45 @@ <h1>Does the person have</h1>
 
       <select id="questions">
         <optgroup label="hair color">
+          <option value=""></option>
```

Comment:
Right now this input value is empty. So the user might not know this is a drop down option meny for where they should choose. A suggestion for change is to put some text in the option tag example: <option value="" selected disabled>Guess here</option>